dem/core/data_management.py

```diff
@@ -23,9 +23,8 @@ def write_deserialized_dev_env_json(dev_env_json_deserialized: dict) -> None:
 # Organization specific dev_env_org.json
 
 def read_deserialized_dev_env_org_json():
-    #Get the raw json file.
-    dev_env_org_json_path = PurePath("example_json/dev_env_org.json")
-    dev_env_org_json = open(dev_env_org_json_path, "r")
-    deserialized_dev_env_org_json = json.load(dev_env_org_json)
-    dev_env_org_json.close()
-    return deserialized_dev_env_org_json
\ No newline at end of file
+    response = requests.get("http://127.0.0.1:8000/dev_env_org.json")
```

Comment:
is that ip ok?

Reply:
Not at all... I added it accidently.
But got fixed in https://github.com/axem-solutions/dem/pull/29.